webhook_features/src/features/summary_comment.rs

```diff
@@ -286,23 +283,25 @@ impl AckType {
     }
 }
 
-macro_rules! multi_vec {
-    ($([$($key:literal),+] => $value:expr);*) => {
-        vec![
-            $($(($key, $value)),*),*
-        ]
-    };
-}
-
 lazy_static! {
-    static ref ACK_PATTERNS: Vec<(&'static str, AckType)> = multi_vec![
-        ["code review ack", "cr ack", "cr-ack", "crack"] => AckType::Ack;
-        ["ack", "utack", "tack"] => AckType::Ack;
-        ["concept ack", "concept-ack", "conceptack", "ack"] => AckType::ConceptAck;
-        ["concept nack", "concept-nack", "conceptnack"] => AckType::ConceptNack;
-        ["approach ack", "approach-ack", "approachack"] => AckType::ApproachAck;
-        ["approach nack", "approach-nack", "approachnack"] => AckType::ApproachNack;
-        ["nack"] => AckType::ConceptNack
+    static ref ACK_PATTERNS: Vec<(&'static str, AckType)> = vec![
+        // Code Review ACK, crack, cr-ack or cr ack [commit hash]
+        (r".*\b(code review ack)((\s)*[0-9a-f]{6,40})\b.*", AckType::Ack),
+        (r".*\b(?:re)?(crack|cr ack|cr-ack)((\s)*[0-9a-f]{6,40})\b.*", AckType::Ack),
```

Comment:
Those two lines can probably be simplified to `\bACK...\b`, then do the parsing of Concept/Approach ACK before.

Only difference would be that "crack" isn't detected, but that isn't a thing anyway, just like "lgtm"

Reply:
I inadvertently removed support for crACK, should I re-add it?

Reply:
Up to you. I don't mind either way